### DISCUSSION TITLE
#Instance Segmentation#PointSup Training demo could not  find the config file

### DISCUSSION THREAD

**Discussion by Shuolongbj** [facebookresearch/detectron2#3263] [General]
Project url:https://github.com/facebookresearch/detectron2/tree/master/projects/PointSup

D:\_DL\detectron2\projects\PointSup>python train_net.py --config-file configs/PointSup/COCO/mask_rcnn_R_50_FPN_3x_point_annotations.yaml --num-gpus 1
Command Line Args: Namespace(config_file='configs/PointSup/COCO/mask_rcnn_R_50_FPN_3x_point_annotations.yaml', dist_url='tcp://127.0.0.1:49153', eval_only=False, machine_rank=0, num_gpus=1, num_machines=1, opts=[], resume=False)
Traceback (most recent call last):
  File "train_net.py", line 108, in <module>
    launch(
  File "d:\_dl\detectron2\detectron2\engine\launch.py", line 82, in launch
    main_func(*args)
  File "train_net.py", line 81, in main
    cfg = setup(args)
  File "train_net.py", line 71, in setup
    cfg.merge_from_file(args.config_file)
  File "d:\_dl\detectron2\detectron2\config\config.py", line 30, in merge_from_file
    assert PathManager.isfile(cfg_filename), f"Config file '{cfg_filename}' does not exist!"
AssertionError: Config file 'configs/PointSup/COCO/mask_rcnn_R_50_FPN_3x_point_annotations.yaml' does not exist!

change the config path 
python train_net.py --config-file configs/PointSup/COCO/mask_rcnn_R_50_FPN_3x_point_annotations_aug.yaml --num-gpus 8
-->
python train_net.py --config-file configs/mask_rcnn_R_50_FPN_3x_point_sup_point_aug_coco.yaml --num-gpus 8

is that right?
@ppwwyyxx

**Comment by ppwwyyxx** [comment]
Thanks and we should fix the instructions in the README
